Makefile

```diff
@@ -297,11 +297,15 @@ $(GOPATH)/bin/goimports:
 	go install golang.org/x/tools/cmd/goimports@v0.1.6
 
 pkg/apis/workflow/v1alpha1/generated.proto: $(GOPATH)/bin/go-to-protobuf $(PROTO_BINARIES) $(TYPES) $(GOPATH)/src/github.com/gogo/protobuf
+	# These files are generated on a v3/ folder by the tool. Link them to the root folder
+	[ -e ./v3 ] || ln -s . v3
```

Comment:
These should not have been removed in https://github.com/argoproj/argo-workflows/pull/6893

Another solution, if creating soft links is not acceptable is to copy the files manually after they are created:

```
...
# These files are generated on a v3/ folder. Move them to the root 
rsync -a -v v3/ ./ --remove-source-files
...
```

I like the link solution better though as it gives the tool access to the existing files if it needs them.